### PR TITLE
Add com.apipost.apipost

### PR DESCRIPTION
**Pull request by jcleng**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  
https://www.apipost.cn/
Apipost=Postman+Swagger+Mock+Jmeter backend, frontend, testing, and online collaboration with real-time content synchronization
Used to predefine interface request parameters and response
      expectations (Mock) before the interface is fully developed, and automatically generate Mock
      URLs, allowing front-end and testing to enter the development process in advance
- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**
https://www.apipost.cn/download.html
There is a public download link available

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by jcleng** [comment]
bot, build com.apipost.apipost

**Comment by jcleng** [comment]
bot, build com.apipost.apipost

**Comment by jcleng** [comment]
bot, build com.apipost.apipost

File: com.apipost.apipost.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "com.apipost.apipost",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
```
Comment (hfiguiere) [inline com.apipost.apipost.json:4]: 24.08 is the current runtime

File: com.apipost.apipost.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Medium.ttc.zip",
+                    "sha256": "d7590637168e1cff4fd3a15422e483308d7ba2a7120ccaf96ead274c3b40e1b3"
+                }
+            ]
+        },
+        {
+            "name": "install",
```
Comment (hfiguiere) [inline com.apipost.apipost.json:81]: please use descriptive names

```suggestion
            "name": "apipost",
```

File: com.apipost.apipost.desktop [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Exec=run.sh
+Icon=com.apipost.apipost
+Name=apipost
+Terminal=false
+Type=Application
+Categories=Development;Network
+Comment=Apipost = Postman + Swagger + Mock + Jmeter 后端、前端、测试，同时在线协作，内容实时同步
```
Comment (hfiguiere) [inline com.apipost.apipost.desktop:8]: this should be in English by default. You can add localizations.

File: com.apipost.apipost.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id type="desktop">com.apipost.apipost</id>
+  <name>apipost</name>
+  <summary>Apipost = Postman + Swagger + Mock + Jmeter 后端、前端、测试，同时在线协作，内容实时同步</summary>
```
Comment (hfiguiere) [inline com.apipost.apipost.metainfo.xml:5]: this should be in English by default.

File: com.apipost.apipost.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://wiki.apipost.cn/docs/introduction</project_license>
```
Comment (hfiguiere) [inline com.apipost.apipost.metainfo.xml:7]: it this redistributable?

File: com.apipost.apipost.json [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 com.apipost.apipost.metainfo.xml -t $FLATPAK_DEST/share/metainfo",
+                "install -Dm644 com.apipost.apipost.desktop /app/share/applications/com.apipost.apipost.desktop",
+                "install -Dm644 com.apipost.apipost.svg /app/share/icons/hicolor/scalable/apps/com.apipost.apipost.svg",
+                "install -Dm644 com.apipost.apipost.png /app/share/icons/hicolor/512x512/apps/com.apipost.apipost.png",
```
Comment (hfiguiere) [inline com.apipost.apipost.json:88]: this is a lie. the icon file is 94x94. Just remove the PNG.

File: com.apipost.apipost.metainfo.xml [hunk 7]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id type="desktop">com.apipost.apipost</id>
```
Comment (hfiguiere) [inline com.apipost.apipost.metainfo.xml:3]: the website is `apipost.cn`. So this is likely incorrect.

**Comment by jcleng** [comment]
Change to there https://github.com/flathub/flathub/pull/5735

**Pull request closed without merging**